code/auth-service/src/auth/datalabsAuthMiddleware.js

```diff
@@ -0,0 +1,11 @@
+import jwt from 'express-jwt';
+import { PUBLIC_KEY } from '../controllers/authorisation';
+
+const datalabsAuthMiddleware = jwt({
+  secret: PUBLIC_KEY,
+  audience: 'https://api.datalabs.nerc.ac.uk/',
```

Comment:
These values are now in the code twice, we might want to extract the config into one place to avoid errors if we change it and forget to change both later.